doc/src/modules/printing.rst

```diff
@@ -307,31 +307,53 @@ This class implements Python printing. Usage::
     x = Symbol('x')
     e = 5*x**3 + sin(x)
 
-srepr
------
+Srepr
+-----------
 
 .. module:: sympy.printing.repr
 
-This printer generates executable code. This code satisfies the identity
-``eval(srepr(expr)) == expr``.
+SymPy doesn’t use repr() for generating textual representation of expressions.
 
-``srepr()`` gives more low level textual output than ``repr()``
+Usage::
 
-Example::
+    >>> repr(5*x**3 + sin(x))
+    '       5*x**3 + sin(x)'
+    >>> expr = exp(-x)*log(1-x)*2
+    >>> repr(expr)
+    '2∗exp(−x)∗log(−x+1)'
+    
 
-  >>> repr(5*x**3 + sin(x))
-  '5*x**3 + sin(x)'
+To get low level textual representation we use function srepr.
 
-  >>> srepr(5*x**3 + sin(x))
-  "Add(Mul(Integer(5), Pow(Symbol('x'), Integer(3))), sin(Symbol('x')))"
+Usage::
 
-``srepr()`` gives the ``repr`` form, which is what ``repr()`` would normally give
-but for SymPy we don’t actually use ``srepr()`` for ``__repr__`` because it’s 
-is so verbose, it is unlikely that anyone would want it called by default.
-Another reason is that lists call repr on their elements, like ``print([a, b, c])``
-calls ``repr(a)``, ``repr(b)``, ``repr(c)``. So if we used srepr for `` __repr__`` any list with 
-SymPy objects would include the srepr form, even if we used ``str()`` or ``print()``.
```

Comment:
These lines are in different place like [this](http://docs.sympy.org/0.7.6/tutorial/printing.html#dot) .Should I add them back ?